docs/install/iam_policy.json

```diff
@@ -65,14 +65,6 @@
             ],
             "Resource": "*"
         },
-        {
-            "Effect": "Allow",
-            "Action": [
-                "ec2:AuthorizeSecurityGroupIngress",
-                "ec2:RevokeSecurityGroupIngress"
```

Comment:
These permissions need scoping down based on the k8s cluster tag or vpc ID, not removed completely. These provide controller permissions to add/remove SG ingress rules on the instance/ENI SGs.